src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayPartKind.cs

```diff
@@ -65,13 +65,19 @@ public enum SymbolDisplayPartKind
         Text = 25,
         /// <summary>The name of a type parameter.</summary>
         TypeParameterName = 26,
-        /// <summary>The name of a query range variable..</summary>
-        RangeVariableName = 27
+        /// <summary>The name of a query range variable.</summary>
+        RangeVariableName = 27,
+        /// <summary>The name of an enum member.</summary>
+        EnumMemberName = 28,
+        /// <summary>The name of an extension method.</summary>
```

Comment:
public docs.  so i would go into a little more detail on this.  Mention that this does *not* appear on extension-method in non-reduced form.